internal/pkg/pct/build_test.go

```diff
@@ -152,7 +140,7 @@ func TestBuild(t *testing.T) {
 					mockError:    nil,
 				},
 			},
-			expectedFilePath:   filepath.Clean("/path/to/nowhere/pkg/template.tar.gz"),
+			expectedFilePath:   filepath.Clean("/path/to/nowhere/template.tar.gz"),
```

Comment:
```suggestion
			expectedFilePath: "/path/to/nowhere/template.tar.gz",
```
filepath is already clean

Reply:
I forgot why I added this originally and removed the `filepath.Clean` based on your suggestion, however this caused the Windows acceptance tests to fail, which then reminded me why I put that in.

I've added a comment for clarification. `filepath.Clean` will ensure that the expected file path is the Windows / *nix version, depending on the platform.